Operations.java

```diff
@@ -0,0 +1,179 @@
+/***
+ * Java Programming Assignment - Week-2, Assignment-2
+ * 
+ * This Program runs the five operations according to the user's choice.
+ * 1. CountPalindromes - Write a Java program that takes a string as input and calculates the number of unique combinations where a palindrome is formed.
+ * 2. NthFibonacci - Write a Java program to print the nth number in the Fibonacci series without using loops.
+ * 3. SnakeToCamel - Write a Java program where the user inputs a string in snake_case. If the string is not in snake_case, first convert it to snake_case, and then to camelCase.
+ * 4. CountConsonants - Write a Java program that takes a string as input and finds the number of consonants in the string. 
+ * 5. BinaryToDecimal - Write a Java program that takes an integer in binary format and converts it to its decimal representation.
+ * 
+ * Program Owner - Keshav Kumar
+ * Date - 11/09/2024
+ * 
+ */
+
+import java.util.Scanner;
+
+public class Operations {
+    static Constant constant = new Constant();

```

Comment:
again constant should all be in uppercase, also use static final since it is supposed to be a constant